helpers/messages.go

```diff
@@ -59,7 +59,8 @@ const (
 	ErrorCodeTeamRequestNotFound      = "Request not found"
 	ErrorCodeTeamMemberNotFound       = "Member not found"
 	ErrorCodeTeamAdminCannotLeave     = "Team administrator cannot leave the team"
-	ErrorCodePrivateTeamJoinForbiden  = "Private Team cannot be joined without consent. Please request an invitation."
+	ErrorCodePrivateTeamJoinForbiden  = "Private Team cannot be joined without consent. Please request an invitation"
+	ErrorCodeTeamRequestAlreadySent   = "Sorry, you already requested an invitation"
```

Comment:
All team errors start with team I think.
could you rename
`ErrorCodePrivateTeamJoinForbiden`  to `ErrorCodeTeamPrivateJoinForbiden`